spec/requests/graphql/query/enabled_dynamic_fields_spec.rb

```diff
@@ -8,30 +8,31 @@
   let(:dynamic_field) { FactoryBot.create(:dynamic_field) }
   let(:default_value) { "exampleDefaultValue" }
   let(:digital_object_type) { "item" }
+  let(:graphql_digital_object_type) { "ITEM" }
```

Comment:
```suggestion
  let(:graphql_digital_object_type) { digital_object_type.upcase }
```
Totally optional suggestion here, but deriving one thing from another is a pattern I often like to follow to reduce the chance of typos.